pkg/events/events.go

```diff
@@ -2950,7 +2950,7 @@ var Definitions = eventDefinitions{
 			ID32Bit: sys32fchownat,
 			Name:    "fchownat",
 			Syscall: true,
-			Sets:    []string{"default", "syscalls", "fs", "fs_file_attr"},
+			Sets:    []string{"syscalls", "fs", "fs_file_attr"},
```

Comment:
fchownat should be kept if we are adding chownat
if we keep chown then fchownat should be kept as well